app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+  const GROUPS = ['control', 'treatment'];
+  const ROLLOUT_CLIENTS = {
+    '3a1f53aabe17ba32': {
+      name: 'Firefox Add-ons',
+      rolloutRate: 0.0 // Rollout rate between 0..1
+    },
+    'dcdb5ae7add825d2': {
+      name: 'Done321',
```

Comment:
this should be 123Done: https://github.com/mozilla/fxa-oauth-server/blob/master/config/dev.json#L11

123Done's evil brother is 321Done I think https://github.com/mozilla/fxa-oauth-server/blob/master/config/dev.json#L30

But Done321 is something that we have yet :)